src/left-nav.jsx

```diff
@@ -41,9 +43,11 @@ const LeftNav = React.createClass({
     className: React.PropTypes.string,
     disableSwipeToOpen: React.PropTypes.bool,
     docked: React.PropTypes.bool,
+    open: React.PropTypes.bool,
```

Comment:
no, it MUST be undefined, to indicate it's not controlled. Unless you want to explicitly document the tri-state-ness of the open prop through that. Is that the case?